python/config_settings/BUILD.bazel

```diff
@@ -10,7 +10,29 @@ filegroup(
     visibility = ["//python:__pkg__"],
 )
 
-construct_config_settings(
-    name = "construct_config_settings",
-    python_versions = TOOL_VERSIONS.keys(),
+string_flag(
+    name = "python_version",
+    # TODO: The default here should somehow match the MODULE config. Until
+    # then, use the empty string to indicate an unknown version. This
+    # also prevents version-unaware targets from inadvertently matching
+    # a select condition when they shouldn't.
+    build_setting_default = "",
+    values = [""] + VERSION_FLAG_VALUES.keys(),
+    visibility = ["//visibility:public"],
 )
+
+[
+    is_python_config_setting(
```

Comment:
Please keep the helper function with the name arg instead of using an inline list comp in the build file. This just makes it more accessible to tooling and keeps all the logic in one place.